### DISCUSSION TITLE
Infinite Loading

### DISCUSSION THREAD

**Discussion by stanislavlysenko0912** [filamentphp/filament#6442] [Help]
Is there a way to fix infinite loading?

Because sometimes not loaded FileUpload component (as in the screenshot "image path") and if it is loaded, it is often the same endless loading when you press "create" button. Who has faced with this? Because I have on different devices, then there is a problem, then no, and I do not want to create Issue for it :)

<img width="317" alt="image" src="https://user-images.githubusercontent.com/86171768/236664157-dc92df64-46ea-4102-8607-47c5bc579e55.png">

```     
Forms\Components\FileUpload::make('image_path')
    ->columnSpanFull()
    ->image()
    ->required()
    ->directory('images/articles')
    ->visibility('public')
```

**Comment by danharrin** [comment]
I havent experienced this. Are there console errors?

**Reply by stanislavlysenko0912** [reply]
<img width="1110" alt="image" src="https://github.com/filamentphp/filament/assets/86171768/a707a895-68b8-4ea9-b65c-b1e0cf86f6aa">
<img width="568" alt="image" src="https://github.com/filamentphp/filament/assets/86171768/6fefbde1-4783-4f88-a3bf-e79edbbe6278">

All network requests have a 200 response, so everything seems to be fine.
Maybe there are some liveware queries to look at, or in what direction is better to look?

**Reply by danharrin** [reply]
Are you on the latest version?

**Reply by stanislavlysenko0912** [reply]
It looks like yes, just wrote `composer update` and `php artisan filament:upgrade` again
```bash
> composer show filament/filament
name     : filament/filament
descrip. : Effortlessly build TALL-powered admin panels.
keywords : 
versions : * v2.17.38
```

I have a hunch that the error is somewhere in this request "livewire/message/app.filament.resources.prompt-resource.pages.create-prompt"

Because the error pops up in the request to liveware, and it only goes 1.
And look, it's not a valid json, there are two

<details>
  <summary>JSON Response</summary>
  
  ```json
{
  "fingerprint": {
    "id": "WAPggYjrvvNTcVNNSHRP",
    "name": "app.filament.resources.prompt-resource.pages.create-prompt",
    "locale": "en",
    "path": "admin/prompts/create",
    "method": "GET",
    "v": "acj"
  },
  "serverMemo": {
    "children": [],
    "errors": [],
    "htmlHash": "9c39507f",
    "data": {
      "record": null,
      "data": {
        "title": null,
        "ai_id": null,
        "type": "text",
        "prompt": null,
        "url": null,
        "image_path": []
      },
      "previousUrl": "https://website.com/admin/prompts",
      "mountedAction": null,
      "mountedActionData": [],
      "componentFileAttachments": [],
      "mountedFormComponentAction": null,
      "mountedFormComponentActionArguments": [],
      "mountedFormComponentActionData": [],
      "mountedFormComponentActionComponent": null
    },
    "dataMeta": [],
    "checksum": "adaf2723d70d327150800329ac951da2262ae280d26225466079175fcfcd60e7"
  },
  "updates": [
    {
      "type": "callMethod",
      "payload": {
        "id": "0ndy",
        "method": "getUploadedFileUrls",
        "params": [
          "data.image_path"
        ]
      }
    }
  ]
}{"effects":{"html":null,"dirty":[]},"serverMemo":{"htmlHash":"00000000","checksum":"53558cabbf6e919cbd50904926e3c890dee6addb7487ecfed0a0b0f592097c33"}}
```
  
</details>

**Reply by danharrin** [reply]
Please create an issue with a reproduction repository
